Classes/Views/DiffString.swift

```diff
@@ -7,6 +7,49 @@
 //
 
 import UIKit
+import Highlightr
+
+func CreateColorCodedString(code: String, includeDiff: Bool=false, lineHandler: ((String, Int) -> LineChangeType)? = nil) -> NSAttributedString {
+    guard let highlightr = Highlightr() else { return NSAttributedString(string: code) }
+
+    highlightr.setTheme(to: "github")
```

Comment:
Note for the future, not part of this card: Would be nice to have the option to switch this theme -- I personally find it easier to read code on a dark background. 🤷‍♂️

Reply:
Adding a new parameter should this trick, right 💃